### PR TITLE
kernel: bump 6.1 to 6.1.75

### PR DESCRIPTION
**Pull request by graysky2**
Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.1.75

Removed upstreamed:
        generic/hack-6.1/321-powerpc_crtsavres_prereq.patch[1]
        generic/pending-6.1/540-ksmbd-only-v2-leases-handle-the-directory.patch[2]

Rebased:
        bcm27xx/patches-6.1/950-0670-fbdev-Don-t-cancel-deferred-work-if-pagelist-empty.patch[3]

All other patches automatically rebased.

1. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.1.75&id=16b88e68b85d5520c28498bb847358ff6300cb90
2. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/fs/smb?h=v6.1.75&id=c866866c795296d3637ab0d48a3d8a3ef5d6f4a3
3. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.1.75&id=81f444f22816a82ada2cc787a2845cffc8c584a6

Build system: x86/64
Build-tested: x86/64/AMD Cezanne
Run-tested: x86/64/AMD Cezanne

### CODE REVIEW

File: target/linux/bcm27xx/patches-6.1/950-0670-fbdev-Don-t-cancel-deferred-work-if-pagelist-empty.patch [hunk 1]
```diff
@@ -24,11 +24,24 @@ Signed-off-by: Phil Elwell <phil@raspberrypi.com>
...
+ 	inode_lock(inode);
+-	flush_delayed_work(&info->deferred_work);
++	/* Kill off the delayed work */
++	cancel_delayed_work_sync(&info->deferred_work);
++
++	/* Run it immediately */
++	schedule_delayed_work(&info->deferred_work, 0);
+ 	inode_unlock(inode);
+ 
+ 	return 0;
```
Comment (DragonBluep) [inline target/linux/bcm27xx/patches-6.1/950-0670-fbdev-Don-t-cancel-deferred-work-if-pagelist-empty.patch:39]: This patch seems can be removed.

**Comment by graysky2** [comment]
I removed it referencing the two upstream changes.

**Comment by antnyfls** [comment]
patch https://github.com/openwrt/openwrt/blob/main/target/linux/bcm27xx/patches-6.1/950-0670-fbdev-Don-t-cancel-deferred-work-if-pagelist-empty.patch has been rebased on the raspberry pi repository 
https://github.com/raspberrypi/linux/commit/cf40d2d8c44a659d09d5c20189aab02f20263e81 .

https://github.com/openwrt/openwrt/blob/main/target/linux/generic/pending-6.1/540-ksmbd-only-v2-leases-handle-the-directory.patch has been upstreamed https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/fs/smb?h=v6.1.75&id=c866866c795296d3637ab0d48a3d8a3ef5d6f4a3

File: target/linux/bcm27xx/patches-6.1/950-0670-fbdev-Don-t-cancel-deferred-work-if-pagelist-empty.patch (hunk 1 above)
Comment (graysky2) [reply target/linux/bcm27xx/patches-6.1/950-0670-fbdev-Don-t-cancel-deferred-work-if-pagelist-empty.patch]: Please see the latest commit which I think gets at your comment.  I will NOT marked this as resolved until I hear back from you.

**Comment by graysky2** [comment]
Thank you @antnyfls - please review my latest commit.

**Comment by lxiaya** [comment]
update 6.1.75

**Comment by schuettecarsten** [comment]
Hmm, my RaspberryPI 4 reboots immdeiately after PPPoE connection to Telekom was established. 

```
Unable to handle kernel access to user memory outside uaccess routines at virtual address 000000000000007a
Mem abort info:
ESR = 0x0000000096000005
EC = 0x25: DABT (current EL), IL = 32 bits
SET = 0, FnV = 0
EA = 0, S1PTW = 0
FSC = 0x05: level 1 translation fault
Data abort info:
ISV = 0, ISS = 0x00000005
CM = 0, WnR = 0
user pgtable: 4k pages, 39-bit VAs, pgdp=000000005d660000
[000000000000007a] pgd=080000005b178003, p4d=080000005b178003, pud=080000005b178003, pmd=0
```

**Pull request merged**